[Bug] eventmesh-connector-redis:test jvm crash when using openjdk8

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.


### Environment

Mac

### EventMesh version

master

### What happened

Task :eventmesh-connector-plugin:eventmesh-connector-redis:test FAILED

 A fatal error has been detected by the Java Runtime Environment:

  SIGSEGV (0xb) at pc=0x000000010642fb52, pid=78526, tid=0x0000000000005c0f

 JRE version: OpenJDK Runtime Environment (8.0_352) (build 1.8.0_352-bre_2022_12_13_23_06-b00)
 Java VM: OpenJDK 64-Bit Server VM (25.352-b00 mixed mode bsd-amd64 compressed oops)
 Problematic frame:
 [thread 3843 also had an error]
V  [libjvm.dylib+0x53cb52]  Unsafe_SetByte+0x61

 Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again

 An error report file with more information is saved as:
 /Users/apple/opensource/incubator-eventmesh/eventmesh-connector-plugin/eventmesh-connector-redis/hs_err_pid78526.log
FAILURE: Build failed with an exception.



### How to reproduce

./gradlew :eventmesh-connector-plugin:eventmesh-connector-redis:test

### Debug logs

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

--- a/eventmesh-connector-plugin/eventmesh-connector-redis/build.gradle
+++ b/eventmesh-connector-plugin/eventmesh-connector-redis/build.gradle
@@ -50,3 +50,7 @@ dependencies {
     testCompileOnly 'org.projectlombok:lombok:1.18.22'
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.22'
 }
+
+test {
+    systemProperty "io.netty.tryUnsafe", "false"
+}

